core/validation_schema.py

```diff
@@ -13,21 +13,20 @@
     },
     "Organisation_Ref": {
         "columns": {
-            "Organisation ID": "str",
             "Organisation": "str",
             "Geography": "str",
         },
-        "uniques": ["Organisation ID", "Organisation"],  # TODO: Assuming we dont want multiple Orgs with the same name
+        "uniques": ["Organisation"],  # TODO: Assuming we dont want multiple Orgs with the same name
```

Comment:
yes, that's correct, can remove the comment

Reply:
removed 👍